resources/sushy-tools/Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM registry.hub.docker.com/library/python:3.7
+FROM registry.hub.docker.com/library/python:3.8
 
 RUN apt update && \
     apt install -y libvirt-dev && \
-    pip3 install sushy-tools==0.10.0 libvirt-python
+    pip3 install git+https://opendev.org/openstack/sushy-tools.git libvirt-python
```

Comment:
Do we need to pick a specific SHA or tag or something to give us some control over when changes make it into the images?

Reply:
sounds like a good idea :)
let's use the sha hash of the change https://review.opendev.org/738874 ?